desktop-style.css

```diff
@@ -178,16 +182,62 @@
     left: 0%;
     width: 100%;
     height: calc(100% - 48px);
-    background: url(images/cards/project_hover.png);
     background-repeat: no-repeat;
     background-size: cover;
+    background-position: center;
   }
 
   .visible {
     visibility: visible;
     display: initial;
   }
 
+  /* popup window */
+
+  .popup {
+    height: 80%;
+    width: 90%;
+    overflow: initial;
+    top: 10%;
+    left: 5%;
+    row-gap: 22px;
+    padding: 64px 24px;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
- [ ] KIndly make sure your popup window in the desktop version is looking the same as the template popup window. Currently, I cannot scroll in your popup window on larges screens size. kindly check what I can see in the desktop version 👇  

Your popup | the template popup
-------|----
![image](https://user-images.githubusercontent.com/65068771/151242565-70cc307a-b93b-42de-9700-b02b0261e474.png) | ![image](https://user-images.githubusercontent.com/65068771/151242790-94e899a4-cd1d-4bba-842a-283cd21d77a4.png)